### PR TITLE
Heading docs fixes

### PR DESCRIPTION
**Pull request by reesscot**
#### Description of changes
Small Heading page fixes:
* Update description to be more consistent with other pages.
* Call out the default heading level in the Levels description
* Update CSS example with new Heading util classes

#### Description of how you validated changes
yarn docs dev

#### Checklist

- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/components/heading/index.page.mdx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: Heading
-description: This is a separate primitive from Text for semantic purposes. The Heading primitive maps to an <h*> tag, and Text maps to a <p> tag.
-themeSource: packages/ui/src/theme/tokens/components/heading.ts
```
Comment (wlee221) [inline docs/src/pages/components/heading/index.page.mdx:4]: For my knowledge, what is `themeSource` and why is it removed here?

**Review by 0618 — APPROVED** [review]

File: docs/src/pages/components/heading/index.page.mdx (hunk 1 above)
Comment (reesscot) [reply docs/src/pages/components/heading/index.page.mdx:4]: It should be removed, thanks for the catch. It's used to render a button that links to the heading design tokens

Comment (reesscot) [reply docs/src/pages/components/heading/index.page.mdx:4]: fixed

**Review by wlee221 — APPROVED** [review]

File: docs/src/pages/components/heading/index.page.mdx (hunk 1 above)
Comment (zchenwei) [reply docs/src/pages/components/heading/index.page.mdx:4]: 😉  You said it should be removed at first

**Review by zchenwei — APPROVED** [review]
LGTM 🚢

File: docs/src/pages/components/heading/index.page.mdx (hunk 1 above)
Comment (wlee221) [reply docs/src/pages/components/heading/index.page.mdx:4]: I catch things without knowing what they are ;)

**Pull request merged**